pkg/tiles/tile.go

```diff
@@ -83,6 +87,15 @@ func (tile Tile) Rotate(rotations uint) Tile {
 	return tile
 }
 
+func (tile *Tile) GetFeatureAtSide(sideToCheck sideMod.Side) *featureMod.Feature {
+	for _, feature := range tile.Features {
+		if sideToCheck&feature.Sides == sideToCheck {
+			return &feature
+		}
+	}
+	return nil
+}
```

Comment:
Will this behave correctly with tiles that have more than one feature on one side?
For example, monastery with a single road has both a road and a field on one side